pkg/collector/corechecks/sbom/processor.go

```diff
@@ -24,10 +29,24 @@ var /* const */ (
 )
 
 type processor struct {
-	queue chan *model.SBOMEntity
+	queue       chan *model.SBOMEntity
+	trivyClient trivy.Collector
+	hostname    string
 }
 
-func newProcessor(sender aggregator.Sender, maxNbItem int, maxRetentionTime time.Duration) *processor {
+func newProcessor(sender aggregator.Sender, maxNbItem int, maxRetentionTime time.Duration) (*processor, error) {
+	trivyConfiguration := trivy.DefaultCollectorConfig([]string{trivy.OSAnalyzers}, "")
+
+	trivyClient, err := trivy.NewCollector(trivyConfiguration)
```

Comment:
If you create a new `TrivyClient` targeting the same cache, we need to make sure that `BoltDB` behind works correctly. It's does not support writers from multiple processes (it uses `flock` for that), not sure what happens with intra-process concurrency and if there's a difference if using one or two instance of `BoltDB` client.

Reply:
Should be fixed by using a single scanner